README.md

```diff
@@ -1 +1,366 @@
-# open-days-registration-application-back-end
\ No newline at end of file
+# General description
+
+The repostiory contains an MVC based [Spring Boot](https://spring.io) REST API application using JAVA programming language and scripts for building a portable [Docker](https://www.docker.com/) container, which runs the application.
+
+You can find more information about the ticket reservation system in this [article](https://db-blog.web.cern.ch/blog/viktor-kozlovszky/2019-10-open-days-reservation-systems-high-level-overview-2019).
+
+The content is orginized into different folders. You will find the description for each folder in this ReadMe.
+
+Overview about the folder structure:
+* **application**: The ticket reservation application's back-end (business logic) implementation. The application receives the API calls from the [front-end server](https://github.com/cerndb/open-days-registration-application-front-end), retrieves or applies modification on the dataset stored in a database.  Please note that the front-end server server has the user interface implementation. It creates http API calls for the back-end server. For more detailed information check out the [ticket reservation application section](https://github.com/cerndb/open-days-registration-application-front-end#Ticket-reservation-application). for making the application running you will need a database server. The current repositry contains the packages connecting to an Oracle database.
+* **containerize**: This folder contains the helper files to build a container which hosts the two applications. For more detailed information check out the container approach section.
+
+Note that the application was using the JDK 1.8 and for building we used Maven 3.6.3. Be aware that currently there is no plan to maintain the code.
```

Comment:
"Be aware that currently there is no plan to maintain the code." I would not add this

Reply:
But that's true, we have added this for the front-end as well.

Reply:
We discussed, we keep it like this